IdentityCore/src/util/NSError+MSIDExtensions.m

```diff
@@ -76,4 +76,17 @@ - (nullable NSString *)msidSubError
     return self.userInfo[self.msidErrorConverter.subErrorKey];
 }
 
+- (nullable NSDictionary*)getStatusCodeUserInfoFromUnderlyingErrorCode
```

Comment:
All category method names should be prefixed with msid, so instead of getStatusCodeUserInfoFromUnderlyingErrorCode -> msidGetStatusCodeUserInfoFromUnderlyingErrorCode

Reply:
Updated